src/tags/templates/tags/detail.html

```diff
@@ -5,7 +5,7 @@
 {% endblock %}
 
 {% block heading_inner %}
-    Tag: {{object.name}}
+    {{object.name}} Tag Details {% if not request.user.is_staff %}for {{request.user.username}}{% endif %}
```

Comment:
Hmmm..,... what if a staff member is viewing a student-specific tag page?  I think this needs more seperation.  url would be something like `/tags/<tag>/<user-id>/` or something like that and the user would be passed in the context (seperately from the viewer / request.user)

Reply:
made a separate view for this
![image](https://user-images.githubusercontent.com/39788517/183555906-d92610b0-09d3-424e-9dbf-9b754c099c34.png)